.github/workflows/buf.yml

```diff
@@ -0,0 +1,27 @@
+name: Buf
```

Comment:
Run `buf` in CI to do lint checks, and in the future, breaking change detection.